utils/functions.py

```diff
@@ -490,7 +490,7 @@ def reconcile_adv(adv=False, dis=False, eadv=False) -> enums.AdvantageType:
     :return: The combined advantage result
     """
     result = 0
-    if adv:
+    if adv or eadv:
```

Comment:
Kind of not actually related to the content of the PR here, but can you add a unit test for this similar to the one in test_argparse for the 8 possible inputs/outputs?